backend/tests/test_address_handlers.py

```diff
@@ -336,7 +339,11 @@ def test_get_mo_address(self):
         }
 
         # Act
-        actual = address_handler.get_mo_address_and_properties()
+        with patch(
+            'mora.service.address_handler.phone.facet.get_one_class',
```

Comment:
Oy vey, that's a long name 😁  Why not do a proper module-level import and just patch `facet.get_one_class`?